### PR TITLE
Allow Crop and CropMirrorNormalize to crop sequences as if they were volumetric images

### PR DESCRIPTION
**Pull request by jantonguirao**
Signed-off-by: Joaquin Anton <janton@nvidia.com>

#### Why we need this PR?
- It adds new feature: 
Allows to use crop_d and crop_z in CropMirrorNormalize and Crop to crop on the sequence dimensions. That is, allows treating 'F' dimension as depth

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
Interpret 'F' dimension as 'D' when crop_d is provided but there is no 'D' dimension in the input layout.
 - Affected modules and functionalities:
Crop and CropMirrorNormalize operators
 - Key points relevant for the review:
Changes in the operators
 - Validation and testing:
New python tests added for both operators
 - Documentation (including examples):
N/A

**JIRA TASK**: *[NA]*

### CODE REVIEW

**Comment by jantonguirao** [comment]
!build

**Comment by jantonguirao** [comment]
!build

File: dali/test/python/test_operator_crop.py [hunk 1]
```diff
@@ -254,6 +254,8 @@ def __init__(self, device, batch_size, iterator, data_shape, data_layout, num_th
             D, H, W = self.data_shape[0], self.data_shape[1], self.data_shape[2]
         elif self.data_layout == "CDHW":
             D, H, W = self.data_shape[1], self.data_shape[2], self.data_shape[3]
+        elif self.data_layout == "FHWC" and crop_seq_as_depth:
```
Comment (JanuszL) [inline dali/test/python/test_operator_crop.py]: Is that possible to have "FCHW" as well?

Comment (jantonguirao) [reply dali/test/python/test_operator_crop.py]: Done

**Comment by jantonguirao** [comment]
!build

**Pull request merged**